backend/spec/features/admin/configuration/payment_methods_spec.rb

```diff
@@ -44,6 +44,17 @@
       click_button "Create"
       expect(page).to have_content("successfully created!")
     end
+
+    it 'disables the button at submit', :js do
+      visit spree.new_admin_payment_method_path
+      page.execute_script "$('form').submit(function(e) { e.preventDefault()})"
+      fill_in "payment_method_name", with: "check90"
+      fill_in "payment_method_description", with: "check90 desc"
+      select Spree::PaymentMethod::Check.model_name.human, from: "Type"
+      click_button "Create"
+
+      expect(find('input[type="submit"]')).to be_disabled
```

Comment:
Could we move this into the other spec instead? We should avoid to have multiple `it` blocks in feature specs. This unnecessarily slows down our test suite